src/components/common/Header/Header.jsx

```diff
@@ -19,17 +22,35 @@ const Header = () => {
   const handleContact = () => {
     setIsContactOpen(!isContactOpen);
   };
+
+  const handleGoBack = () => {
+    history.goBack();
+  };
+
   return (
-    <header className="header">
-      <div
-        className="header__icon-container"
-        role="button"
-        onClick={handleClick}
-        onKeyPress={handleClick}
-        tabIndex={0}
-      >
-        <img src={isOpen ? close : hamburger} alt="menu icon" className="header__icon" />
-      </div>
+    <header className={`header ${location.pathname == '/new-target' ? 'bg-lb' : 'bg-w'}`}>
+      {location.pathname === '/new-target' ? (
+        <div className="header__actions">
+          <img
+            src={back}
+            alt="go back"
```

Comment:
Since you are using i18n, WDYT if you apply it on image's alts?

Reply:
I don't understand what you mean